aliases/git/nu_alias_git.nu

```diff
@@ -0,0 +1,202 @@
+export alias gbs                   = git bisect
+export alias gbsb                  = git bisect bad
+export alias gbsg                  = git bisect good
+export alias gbsr                  = git bisect reset
+export alias gbss                  = git bisect start
+
+export alias gc                    = git commit -v
+export alias gc!                   = git commit -v --amend
+export alias gcn                   = git commit -v --no-edit
+export alias gcn!                  = git commit -v --no-edit --amend
+export alias gca                   = git commit -v -a
+export alias gca!                  = git commit -v -a --amend
+export alias gcan!                 = git commit -v -a --no-edit --amend
+export alias gcans!                = git commit -v -a -s --no-edit --amend
+export alias gcam                  = git commit -a -m
+export alias gcsm                  = git commit -s -m
+export alias gcas                  = git commit -a -s
+export alias gcasm                 = git commit -a -s -m
+export alias gcb                   = git checkout -b
+export alias gcf                   = git config --list
+
+export alias gcl                   = git clone --recurse-submodules
+export alias gclean                = git clean -id
+export alias gpristine             = ((git reset --hard); (git clean -dffx))
+export alias gcm                   = git checkout $(git_main_branch)
+export alias gcd                   = git checkout $(git_develop_branch)
+export alias gcmsg                 = git commit -m
+export alias gco                   = git checkout
+export alias gcor                  = git checkout --recurse-submodules
+export alias gcount                = git shortlog -sn
+export alias gcp                   = git cherry-pick
+export alias gcpa                  = git cherry-pick --abort
+export alias gcpc                  = git cherry-pick --continue
+export alias gcs                   = git commit -S
+export alias gcss                  = git commit -S -s
+export alias gcssm                 = git commit -S -s -m
+
+export alias gd                    = git diff
+export alias gdca                  = git diff --cached
+export alias gdcw                  = git diff --cached --word-diff
+export alias gdct                  = git describe --tags $(git rev-list --tags --max-count=1)
+export alias gds                   = git diff --staged
+export alias gdt                   = git diff-tree --no-commit-id --name-only -r
+export alias gdup                  = git diff @{upstream}
+export alias gdw                   = git diff --word-diff
+
+export alias gf                    = git fetch
+export alias gfo                   = git fetch origin
+
+export alias gfg                   = (git ls-files | grep)
+
+export alias gg                    = git gui citool
+export alias gga                   = git gui citool --amend
+
+export alias ggpur                 = ggu
+export alias ggpull                = git pull origin "$(git_current_branch)"
```

Comment:
does this work? It looks like bash syntax vs nushell syntax.

Reply:
whops. No this line shouldn't work. It also makes use of an old function which I removed ;-;